sequence_processing_pipeline/templates/nuqc_job.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash -l
+#SBATCH -J {{job_name}}
+# wall-time-limit in minutes
+#SBATCH --time {{wall_time_limit}}
+#SBATCH --mem {{mem_in_gb}}gb
+#SBATCH -N {{node_count}}
+#SBATCH -c {{cores_per_task}}
+#SBATCH --output %x-%A_%a.out
+#SBATCH --error %x-%A_%a.err
```

Comment:
For this to work fine, this sh needs to be run from the output folder, right? Just a question, fine either way.

Reply:
If a different location for stdout/stderr is needed, can't the path be added in front of `%x`?

Reply:
I don't know anything about how qiita manages slurm stdout/stderr, I think this is a detail that you need to decide on

Reply:
I added a line to run it in the same output folder all our other job scripts run in e.g: <working-directory>/ConvertJob, <working-directory>/QCJob. I also removed the output and error switches because Qiita and Slurm handle logging for us so these shouldn't be needed; they aren't found in our other job scripts. Lastly I added a few debugging echos in line with our other scripts.